components/infobox/wikis/overwatch/infobox_map_custom.lua

```diff
@@ -0,0 +1,74 @@
+---
+-- @Liquipedia
+-- wiki=overwatch
+-- page=Module:Infobox/Map/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Map = require('Module:Infobox/Map')
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local MapModes = require('Module:MapModes')
+local String = require('Module:StringUtils')
+
+local CustomMap = Class.new()
+
+local CustomInjector = Class.new(Injector)
+
+local _args
+
+function CustomMap.run(frame)
+	local customMap = Map(frame)
+	customMap.createWidgetInjector = CustomMap.createWidgetInjector
+	customMap.addToLpdb = CustomMap.addToLpdb
+	_args = customMap.args
+	return customMap:createInfobox(frame)
+end
+
+function CustomMap:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:addCustomCells(widgets)
+	local args = _args
```

Comment:
don't think if it necessary to define args from _args in this function since args is only used one in the function
so maybe kick this line and change `args` to `_args` in line 42